main/uart_nic.c

```diff
@@ -250,7 +232,9 @@ static void event_handler(void* arg, esp_event_base_t event_base, int32_t event_
    }
 }
 
-static int IRAM_ATTR wifi_receive_cb(void *buffer, uint16_t len, void *eb) {
+static int wifi_receive_cb(void *buffer, uint16_t len, void *eb) {
+    esp_wifi_internal_free_rx_buffer(eb);
```

Comment:
I'm not entirely sure what this thing does. But to double check ‒ is it OK to free this first and then access the data?